Lib/argparse.py

```diff
@@ -264,7 +264,7 @@ def add_argument(self, action):
                 invocations.append(get_invocation(subaction))
 
             # update the maximum item length
-            invocation_length = max([len(s) for s in invocations])
+            invocation_length = max(map(len, invocations))
```

Comment:
@rhettinger Looks awesome, is there any performance difference?

Reply:
It was either this or a switch to a generation expression to save memory.  Depending on the number of invocations, the *map* version has fewer global lookups and fewer turns around the ceval loop.  The win is likely very minor though.  Mostly, it reads a little better than before.

Reply:
Thanks for the explanation :)